.github/workflows/build.yml

```diff
@@ -89,7 +104,7 @@ jobs:
           java-version: "adopt@1.8"
       - name: Add coverage to PR
         id: jacoco
-        uses: madrapps/jacoco-report@v1.5
+        uses: madrapps/jacoco-report@v1.6.1
```

Comment:
if you want to be consistent with https://github.com/AbsaOSS/atum-service/pull/185, consider to extract this version elsewhere

Reply:
The version is used once only. The rest of the file do not follow this. In the atum-service project, this change produces an error.